packages/maps/src/components/ChatbotIframe.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { types } from "@probable-futures/lib";
+
+import ChatbotIcon from "../assets/icons/chatbot.svg";
+import { sendDataToChatbot } from "../utils/chatbot";
+
+type IFrameProps = { selectedData?: types.Map; degrees?: number };
+
+type ButtonProps = {
+  position?: string;
+  top?: string;
+  right?: string;
+};
+
+const StyledButton = styled.button`
+  ${({ position, top, right }: ButtonProps) => `
+    border: none;
+    background-color: transparent;
+    border-radius: 50%;
+    cursor: pointer;
+    position: ${position};
+    top: ${top};
+    right: ${right};
+  `}
+`;
+
+const StyledIframe = styled.iframe`
+  width: 480px;
+  height: 570px;
+  border: none;
+`;
+
+const ChatIcon = styled.img`
+  width: 54px;
+  height: 54px;
+`;
+
+const ChatbotIframe = ({ selectedData, degrees }: IFrameProps) => {
+  const [isChatOpen, setIsChatOpen] = useState(false);
+
+  const toggleChat = () => {
+    setIsChatOpen(!isChatOpen);
+  };
+
+  const chatbotUrl = process.env.REACT_APP_CHATBOT_URL;
```

Comment:
move outside the component function

Reply:
[commit](https://github.com/Probable-Futures/apps-public/pull/1/commits/9c2655769af801d94fff5c94001412cfbf752777)